Added Branding color, logo and favicon

jnkindi opened the pull request:
- Changed main color to #ff5c00
- Modified Header Logo from Text to Logo
- Changed favicon

**Before**
<img width="1239" alt="Screen Shot 2020-09-02 at 13 50 16" src="https://user-images.githubusercontent.com/14198719/91977682-4d9d1580-ed23-11ea-8833-bb978e84afa3.png">


**After**
<img width="1240" alt="Screen Shot 2020-09-02 at 13 50 29" src="https://user-images.githubusercontent.com/14198719/91977706-542b8d00-ed23-11ea-8a80-034b14751d95.png">

davidlesieur commented:
Minor comments regarding these changes:

* I have not seen the new design guidelines, but it seems strange to me that the logo image has so much whitespace around the actual logo. It causes a horizontal misalignment with respect to other titles in the Explore column. If such whitespace is nevertheless desired, I'd rather use CSS margin or padding to retain more layout control on various media sizes. Also, the tagline's text seems very small, but then again I have not seen the new design guidelines.
* If the "custom-logo" class is not used, I'd remove it from the `<img>` tag.
* `scale` is not a valid attribute for `<img>` tags.

I like the new branding color, more contrasty, more accessible!

jnkindi commented:
@davidlesieur , Thanks for the feedback.
I have implemented these changes:
- Removed "custom-logo" class on logo img
- Removed the scale attribute on logo img
- I trimmed the logo to remove the whitespace around the actual logo
